### PR TITLE
Fix translation issues

### LINKED ISSUE #2053: German Ingame-Translation is broken in 0.4.11
bildschirmfoto 2015-01-05 um 13 30 25
bildschirmfoto 2015-01-05 um 13 31 26

in the launcher is german text.

bildschirmfoto 2015-01-05 um 13 25 59

### PR DESCRIPTION
**Pull request by ShadowNinja**
Fixes #2053 and #2174.

### CODE REVIEW

**Comment by ngosang** [comment]
I tested this changes and all is working fine! Thank you ShadowNinja. This pull #2169 should complete all remaining translations.

**Comment by ShadowNinja** [comment]
Rebased.  I also tried to reduce the `string` / `wstring` / `char*` / `wchar_t*` conversions.  This should be tested on Windows too though.

File: src/gettext.h [hunk 1]
```diff
@@ -43,54 +43,30 @@ with this program; if not, write to the Free Software Foundation, Inc.,
...
-	nstr = new wchar_t[l+1];
-
-	std::wstring intermediate = narrow_to_wide(str);
-	memset(nstr, 0, (l+1)*sizeof(wchar_t));
-	memcpy(nstr, intermediate.c_str(), l*sizeof(wchar_t));
-#endif
-
-	return nstr;
+// You must free the returned string!
+inline const wchar_t *wgettext(const char *str) {
```
Comment (rubenwardy) [inline src/gettext.h]: Braces on functions in Minetest start on a new line. :P

**Comment by nerzhul** [comment]
@ShadowNinja could you rebase another time ?

**Comment by ShadowNinja** [comment]
@nerzhul This was merged, the merger forgot to close this.

**Pull request closed without merging**